staging/src/k8s.io/apimachinery/pkg/apis/meta/internalversion/types.go

```diff
@@ -35,6 +35,14 @@ type ListOptions struct {
 	FieldSelector fields.Selector
 	// If true, watch for changes to this list
 	Watch bool
+	// allowBookmarks requests watch events with type "BOOKMARK".
```

Comment:
nit: `AllowBookmarks` (casing should be the same as field).

Reply:
Call this `AcceptsBookmarks` ? The protocol here is similar to http content negotiation so 'accepts' might be a more familiar term for the purpose of the field..

Reply:
The API review is happening here: https://github.com/kubernetes/kubernetes/pull/74074